hera_cal/delay_filter.py

```diff
@@ -180,6 +282,9 @@ def delay_filter_argparser():
     a.add_argument("--CLEAN_outfilename", default=None, type=str, help="path for writing the CLEAN model visibilities (with the same flags)")
     a.add_argument("--filled_outfilename", default=None, type=str, help="path for writing the original data but with flags unflagged and replaced with CLEAN models wherever possible")
     a.add_argument("--clobber", default=False, action="store_true", help='overwrites existing file at outfile')
+    a.add_argument("--spw_range", type=int, nargs=2, help="spectral window of data to foreground filter.")
```

Comment:
If you're going to have multiple run scripts, you should have multiple argparsers or do something like: https://github.com/HERA-Team/hera_qm/blob/8ed3a97f9e37326295bf97df143ec803255b3d2c/hera_qm/utils.py#L24

As it stands, if you run the scripts with `-h` you'll get documentation for parameters that aren't used and aren't relevant, which is confusing.

Reply:
I've split this parser up into several that are appropriate for different scripts.